bin/asl-say

```diff
@@ -131,10 +148,12 @@ function usage(){
 	exit 1
 }
 
-while getopts "n:w:" opt; do
+while getopts "n:w:d:t:" opt; do
 	case ${opt} in
 		n)	NODE="${OPTARG}" ;;
 		w)	WHAT="${OPTARG}" ;;
+		d)	ALT_DATE="${OPTARG}" ;;
+		t)	ALT_TIME="${OPTARG}" ;;
```

Comment:
I'm a bit  unsure why we need to split this into a date and a time component when the `date` function takes a single argument. For instance, if I do `date -d " 8:15"` it already returns a datetime representing fifteen minutes past eight on the current day. See other notes below.

Reply:
Interesting.  I was looking that the date(1) man page on my Mac ... that does not have the "-d" option.  That option, available on Debian, makes life much simpler.

Reply:
This code has been reworked